README.md

```diff
@@ -45,16 +74,36 @@ The nf-core/viralrecon pipeline comes with documentation about the pipeline, fou
 2. Pipeline configuration
     * [Local installation](https://nf-co.re/usage/local_installation)
     * [Adding your own system config](https://nf-co.re/usage/adding_own_config)
-    * [Reference genomes](https://nf-co.re/usage/reference_genomes)
+    * [Reference genomes](docs/usage.md#reference-genomes)
 3. [Running the pipeline](docs/usage.md)
 4. [Output and how to interpret the results](docs/output.md)
 5. [Troubleshooting](https://nf-co.re/usage/troubleshooting)
 
-<!-- TODO nf-core: Add a brief overview of what the pipeline does and how it works -->
-
 ## Credits
 
-nf-core/viralrecon was originally written by Sarai Varona and Sara Monzon.
+nf-core/viralrecon was originally written by Sarai Varona, Miguel Juliá and Sara Monzon from [BU-ISCIII](https://github.com/BU-ISCIII) and co-ordinated by Isabel Cuesta for the [Institute of Health Carlos III](https://eng.isciii.es/eng.isciii.es/Paginas/Inicio.html), Spain. Through collaboration with the nf-core community the pipeline has now been updated substantially to include additional processing steps, to standardise inputs/outputs and to improve the pipeline reporting.
```

Comment:
to improve the pipeline reporting &rarr; to improve pipeline reporting

Reply:
Fixed https://github.com/nf-core/viralrecon/pull/106/commits/20ca868fa02830101d6d513287c572173146dd35